cocos/gfx/webgl/webgl-device.ts

```diff
@@ -110,7 +128,6 @@ export class WebGLDevice extends Device {
     private _context: WebGLRenderingContext | null = null;
     private _bindingMappings: IWebGLBindingMapping | null = null;
 
-    /** @mangle */
```

Comment:
`WebGLDevice` is mangled, so all properties in it will be mangled, so it isn't need here.